src/DXRModules/DeusEx/Classes/DXRStartMap.uc

```diff
@@ -716,6 +716,52 @@ function DeusExNote AddNoteFromConv(#var(PlayerPawn) player, bool bEmptyNotes, n
     return None;
 }
 
+#ifdef transcended
+function DeusExNote AddNoteFromText(#var(PlayerPawn) player, bool bEmptyNotes, name textName, optional string textpackage, optional bool bRemoveBlankLines, optional bool bUserNote, optional bool bShowInLog, optional string strSource)
+#else
+function DeusExNote AddNoteFromText(#var(PlayerPawn) player, bool bEmptyNotes, name textName, optional string textpackage, optional bool bRemoveBlankLines, optional bool bUserNote, optional bool bShowInLog)
+#endif
+{
+	local DeusExTextParser parser;
+    local string note, text;
+    local DeusExNote dxNote;
+
+    if (bEmptyNotes == false) {
+        return None;
+    }
+    parser = new(None) Class'DeusExTextParser';
+    if (parser == None) return None;
+
+    parser.OpenText(textName, textpackage);
+    while(parser.ProcessText()) {
+        text = parser.GetText();
+        if (bRemoveBlankLines && text == "  ") {
+            parser.ProcessText();
+            continue;
+        }
+
+        if (text == "") {
+            note = note $ "|n";
+        } else {
+            note = note $ text;
+        }
+    }
+    CriticalDelete(parser);
```

Comment:
The text should be closed before deleting the parser object.  This may leave a dangling file reference or something.  Not sure how bad this may be, exactly, but it should be done if only for correctness.